addon/services/popover.js

```diff
@@ -131,7 +131,7 @@ export default Ember.Service.extend({
     };
 
     popoverSpec.childViews = [
-      ComponentClass.create({
+      ComponentClass.extend({
```

Comment:
This change is required because the `{{view}}` helper sets up the `container`, `ownerView`, etc for View classes not instances, unlike a ContainerView's `childViews`.

This was causing a few tests too failing when using the deprecated API.